packages/cyberstorm/src/components/Layout/ErrorLayout/ErrorLayout.tsx

```diff
@@ -0,0 +1,44 @@
+import styles from "./ErrorLayout.module.css";
+import { BaseLayout } from "../BaseLayout/BaseLayout";
+
+export interface ErrorLayoutProps {
+  error: 404 | 500;
+}
+
+/**
+ * Cyberstorm NotFound Layout
+ */
+export function ErrorLayout(props: ErrorLayoutProps) {
+  const { error } = props;
+
+  let errorCode = "Error";
+  let errorDescription = "Unknown";
+  let errorFlavorText = "An error with an unknown error code happened";
+
+  if (error === 404) {
+    errorCode = "404";
+    errorDescription = "Page not found";
+    errorDescription = "Page not found";
```

Comment:
I got it the first time :trollface:

Reply:
Ctrl + D and Ctrl + S are dangerously close to each other xD